src/ripple/app/paths/PathRequest.cpp

```diff
@@ -380,7 +424,33 @@ int PathRequest::parseJson (Json::Value const& jvParams, bool complete)
                 uIss = *raSrcAccount;
             }
 
-            sciSourceCurrencies.insert ({uCur, uIss});
+            if (raSrcAccount && saSendMax)
```

Comment:
Does this mean that send_max is basically ignored if there is no source_account? Should that case be a parsing error?